packages/calcite-design-tokens/support/token-transformer/styleDictionary/parser/utils/resolveRelevance.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * This is a copy/extension of sd-transforms/src/parsers/resolveReference.ts but now with better types!
```

Comment:
Can you make sure all copies have a note on them? I think we need a way to separate them. 

How will we keep these in sync with the upstream source?

Reply:
We don't have to. I pulled in the fuctions they use that I think are helpful but since design is moving away from Token Studio we should not need to keep track of the upstream source.